### PR TITLE
santize user input in ExeCmdXXX for show.go

### PR DESCRIPTION
**Pull request by neo-liang-sap**
**What this PR does / why we need it**:
santize user input in ExeCmdXXX for show.go
**Which issue(s) this PR fixes**:
**Special notes for your reviewer**:
@petersutter 
**Release note**:
```improvement operator

```

### CODE REVIEW

File: pkg/cmd/show.go [hunk 1]
```diff
@@ -407,7 +407,7 @@ func showTerraform(name string) {
 	count := 0
 	for _, pod := range pods.Items {
 		if strings.Contains(pod.Name, name) && pod.Status.Phase == "Running" {
-			output, err = ExecCmdReturnOutput("bash", "-c", "export KUBECONFIG="+KUBECONFIG+"; kubectl get pods "+pod.Name+" -o wide -n "+pod.Namespace)
+			output, err = ExecCmdReturnOutput("kubectl", "--kubeconfig="+KUBECONFIG, "get", "pods", pod.Name, "-o wide", "-n "+pod.Namespace)
```
Comment (petersutter) [inline pkg/cmd/show.go:410]:
```go
output, err = ExecCmdReturnOutput("kubectl", "--kubeconfig="+KUBECONFIG, "get", "pods", pod.Name, "-o wide", "-n "+pod.Namespace)
```
I guess this won't work, right?
either
- ` "-owide", "-n"+pod.Namespace` (no space)
- ` "-o", "wide", "-n", pod.Namespace` (or separate argument with comma)

Comment (neo-liang-sap) [reply pkg/cmd/show.go]: yes you are right, thanks! i fixed this and also include some missing parts

File: pkg/cmd/show.go [hunk 2]
```diff
@@ -345,7 +345,7 @@ func showKubernetesDashboard(targetReader TargetReader) {
 func showGrafana(targetReader TargetReader) {
 	username, password = getMonitoringCredentials()
 	showPod("grafana", "seed", targetReader)
-	output, err := ExecCmdReturnOutput("bash", "-c", "export KUBECONFIG="+KUBECONFIG+"; kubectl get ingress grafana -n "+getTechnicalID())
+	output, err := ExecCmdReturnOutput("kubectl", "--kubeconfig="+KUBECONFIG, "get", "ingress", "grafana", "-n"+getTechnicalID())
```
Comment (petersutter) [inline pkg/cmd/show.go:348]:
```suggestion
	output, err := ExecCmdReturnOutput("kubectl", "--kubeconfig="+KUBECONFIG, "get", "ingress", "grafana", "-n", getTechnicalID())
```
your change also works, but I guess we should be more consistent with the arguments (`"-n"+getTechnicalID()` vs `"-n", getTechnicalID()`). Same applies for the other occurences

File: pkg/cmd/show.go [hunk 3]
```diff
@@ -134,7 +134,7 @@ func showOperator() {
 // showUI opens the gardener landing page
 func showGardenerDashboard() {
 	showPodGarden("gardener-dashboard", "garden")
-	output, err := ExecCmdReturnOutput("bash", "-c", "export KUBECONFIG="+KUBECONFIG+"; kubectl get ingress gardener-dashboard-ingress -n garden")
+	output, err := ExecCmdReturnOutput("kubectl", "--kubeconfig="+KUBECONFIG, "get", "ingress", "gardener-dashboard-ingress", "-n"+getTechnicalID())
```
Comment (petersutter) [inline pkg/cmd/show.go:137]: I'm not sure if the namespace is correct. The gardener dashboard is usually in the garden namespace

Comment (neo-liang-sap) [reply pkg/cmd/show.go]: thanks! it's my typo and fixed

File: pkg/cmd/show.go (hunk 2 above)
Comment (neo-liang-sap) [reply pkg/cmd/show.go]: thanks, i also applies this to other occurence

Comment (petersutter) [reply pkg/cmd/show.go]: You have missed:
> Same applies for the other occurences

**Review by petersutter — APPROVED** [review]
/lgtm, thanks

**Comment by neo-liang-sap** [comment]
thanks @petersutter , i will squash and merge

**Pull request merged**